packages/vee-validate/src/Field.ts

```diff
@@ -225,3 +230,11 @@ function resolveValidationTriggers(props: ValidationTriggersProps) {
     validateOnModelUpdate: props.validateOnModelUpdate ?? validateOnModelUpdate,
   };
 }
+
+function applyModifiers(value: any, modifiers: Record<string, boolean>) {
```

Comment:
I would like to limit the usage of `any` when possible, so let's use `unknown` here

```suggestion
function applyModifiers(value: unknown, modifiers: Record<string, boolean>) {
```